ocean_lib/services/service.py

```diff
@@ -272,3 +301,79 @@ def update_compute_values(
         ] = trusted_algo_publishers
         self.compute_values["allowNetworkAccess"] = allow_network_access
         self.compute_values["allowRawAlgorithm"] = allow_raw_algorithm
+
+
+class ConsumerParameters:
```

Comment:
I think it would be good to move this to it's own file, `consumer_parameters.py`